src/Target.cpp

```diff
@@ -801,6 +817,10 @@ std::string Target::to_string() const {
     if (has_feature(Target::TraceLoads) && has_feature(Target::TraceStores) && has_feature(Target::TraceRealizations)) {
         result = Internal::replace_all(result, "trace_loads-trace_realizations-trace_stores", "trace_all");
     }
+    if (vector_bits != 0) {
+        result += "vector_bits:" + std::to_string(vector_bits);
```

Comment:
Not sure I understand point 1. Are you saying it needs to emit something when vector_bits is 0?

It is not intended to be a feature flag and I thought it always came last. Ideally I'd have added a different syntax, but attempts to do that broke many things. (E.g. adding the idea of "foo=bar" parameters.)

Reply:
The colon is left over from a previous syntax. I thought I had test coverage for that.

Reply:
Fixed the formatting and added test coverage.

The idea here is that this is not a feature flag because it doesn't show up int he feature flags set. I feel like any code dealing with it needs to know that it can have an arbitrary number inside it anyway, so the fact that it canonicalizes to the end is fine, and perhaps a good signal that it is a bit different.

One issue is this doesn't play well with the can_use_target features machinery. We'll have to extend that anyway to handle an arbitrary numeric parameter. (And I really don't want to allocate a large number of feature flags to represent all the likely sizes.)